### PR TITLE
Add support for PHP 8

### PR DESCRIPTION
**Pull request by lepiaf**

### CODE REVIEW

**Comment by lepiaf** [comment]
```
composer update ${COMPOSER_FLAGS} --prefer-dist --no-interaction

Loading composer repositories with package information

Updating dependencies

Your requirements could not be resolved to an installable set of packages.

  Problem 1

    - php-translation/common[3.0.0, ..., 3.0.1] require php ^7.2 -> your php version (8.0.0) does not satisfy that requirement.

    - Root composer.json requires php-translation/common ^3.0 -> satisfiable by php-translation/common[3.0.0, 3.0.1].

The command "composer update ${COMPOSER_FLAGS} --prefer-dist --no-interaction" failed and exited with 2 during .
```

I've opened a PR https://github.com/php-translation/common/pull/40

**Comment by bocharsky-bw** [comment]
Thank you for working on it!

As soon as https://github.com/php-translation/common/pull/36 is merged and the package is tagged with v4.0.0 - I think we can merge this one too, but we will need to tweak `php-translation/common` to `^3.0 || ^4.0` here

File: composer.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
         }
     ],
     "require": {
-        "php": "^7.2",
-        "php-translation/common": "^3.0",
+        "php": "^7.2 || ^8.0",
+        "php-translation/common": "^3.0 || ^4.0",
```
Comment (bocharsky-bw) [inline composer.json]: I think it's still too early for allowing 4.0, we will tag v3.1.0 first, see https://github.com/php-translation/common/pull/43

**Review by Nyholm — APPROVED** [review]
Awesome. Thank you for the work!

**Pull request merged**